contracts/contracts/strategies/ConvexStrategy.sol

```diff
@@ -18,12 +18,6 @@ contract ConvexStrategy is BaseCurveStrategy {
     using StableMath for uint256;
     using SafeERC20 for IERC20;
 
-    event RewardTokenCollected(
```

Comment:
I like that everything uses the same event now.

Reply:
😊